include/psp2/types.h

```diff
@@ -7,6 +7,8 @@
 #ifndef _PSP2_TYPES_H_
 #define _PSP2_TYPES_H_
 
+#define PSP2_SDK_VERSION 0x03570011
```

Comment:
how about just make the another header like `def.h`?

Reply:
With a single define?

Reply:
yep. imo, a version couldn't be types.